wagtail/wagtailadmin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -417,19 +417,76 @@ footer,
             display: none;
         }
 
-        img {
-            width: 60px;
+        .wagtail-logo-container {
             float: none;
             margin: auto;
             display: block;
+            width: 60px;
+            height: 75px;
+            position: relative;
+            transition: all 0.25s cubic-bezier(0.28, 0.15, 0, 2.1);
+
+            img {
```

Comment:
The container is required to rotate the wagtail during the 'nod' animation, perhaps there is another approach I could take for this?